sql/4_2_2-to-5_0_0_upgrade.sql

```diff
@@ -2137,3 +2141,21 @@ CREATE TABLE `product_registration` (
 #IfNotRow2D list_options list_id Eye_Defaults_for_GENERAL option_id LADNEXA
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`) VALUES ('Eye_Defaults_for_GENERAL', 'LADNEXA', 'normal lacrimal gland and orbit', 91, 0, 0, '', 'EXT', '', 0, 0, 0, '');
 #EndIf
+
+-- Table to copy log contents for audit log tamper resistance check
+#IfNotTable log_validator
+CREATE TABLE `log_validator` (
+  `log_id` bigint(20) NOT NULL,
+  `log_checksum` longtext DEFAULT NULL,
```

Comment:
remove default NULL here(this is not legal in newer mysql versions):
``````
`log_checksum` longtext,